snarkyjs/index.js

```diff
@@ -78,9 +78,30 @@ module.exports = exports = function (name) {
         });
       }, function (err) { return err; });
   };
+  var generate_keys = function() {
+    var query_json = JSON.stringify({"command": "verify"});
```

Comment:
Pretty sure the command here should be "generate_keys".

Reply:
Good catch! Fixed.